.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+package.json
+package-lock.json
```

Comment:
- Please, kindly remove the `package and packag-lock.json` file as you need them to be visible so that other users can see the packages you've used in building the project.

Reply:
## MISSING LINTER FILE THAT NEEDS TO BE ADDED

- Hey, while you've done great by adding the linter file, please kindly add the `.hintrc` and `.stylelintrc.json` from the microverse [HTML and CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linter file.
- After adding them, do ensure there are no linter errors.

Reply:
noted changes done